lib/Conversion/ExportVerilog/ExportVerilog.cpp

```diff
@@ -5186,6 +5186,61 @@ void ModuleEmitter::emitHWModule(HWModuleOp module) {
   currentModuleOp = HWModuleOp();
 }
 
+/// Create a MacroDeclOp and MacroDefOp for the ExportableRefOp.
+/// This is called before the symbolCache has been frozen,
+/// so the MacroDefOp will not have it's format string set yet.
+static MacroDefOp createExportableRefMacro(ExportableRefOp op) {
```

Comment:
ExportVerilog shouldn't be creating new macros, it should be emitting macros that already exist.